src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -27,22 +29,30 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html");
-
     UserService userService = UserServiceFactory.getUserService();
-    if (userService.isUserLoggedIn()) {
-      String userEmail = userService.getCurrentUser().getEmail();
-      String urlToRedirectToAfterUserLogsOut = "/login";
-      String logoutUrl = userService.createLogoutURL(urlToRedirectToAfterUserLogsOut);
+    boolean loginStatus = userService.isUserLoggedIn();
+    String urlToRedirect = "/index.html";
 
-      response.getWriter().println("<p>Hello " + userEmail + "!</p>");
-      response.getWriter().println("<p>Logout <a href=\"" + logoutUrl + "\">here</a>.</p>");
+    ArrayList<String> userDetails = new ArrayList();
```

Comment:
`userDetails` makes me think that this will have user info in it but it looks like what's actually added below is the html for the response. Perhaps a more descriptive variable name?

Reply:
I just added email and school name- hopefully that makes more sense? if not, I can change it to loginDetails. I intend for this to later hold the user's name and other facts that they might put in (year in school, etc)